hedera-node/hedera-mono-service/src/test/java/com/hedera/services/store/contracts/precompile/PrngSystemPrecompiledContractTest.java

```diff
@@ -305,7 +318,10 @@ void parentUpdaterMissingFails() throws InterruptedException {
         given(updater.parentUpdater()).willReturn(Optional.empty());
         given(creator.createSuccessfulSyntheticRecord(anyList(), any(), anyString()))
                 .willReturn(childRecord);
-        given(frame.getBlockValues()).willReturn(new HederaBlockValues(10L, 123L, consensusNow));
+        given(frame.getBlockValues())
+                .willReturn(
+                        new HederaBlockValues(
+                                10L, 123L, MiscUtils.timestampToInstant(consensusNow)));
         given(runningHashLeaf.nMinusThreeRunningHash())
                 .willReturn(new Hash(TxnUtils.randomUtf8Bytes(48)));
```

Comment:
There shouldn't be any test changes, since no changes should be required for `PrngSystemPrecompileContract`.